BitwardenShared/UI/Vault/VaultItem/ViewItem/ViewItemProcessor.swift

```diff
@@ -443,6 +515,7 @@ private extension ViewItemProcessor {
                 }
                 newState.hasVerifiedMasterPassword = state.hasVerifiedMasterPassword
                 state = newState
+                try await services.eventService.collect(eventType: .cipherClientViewed, cipherId: itemId)
```

Comment:
🤔 This is going to get called again any time the cipher is updated while on the view screen. Is that expected? Otherwise, maybe this could be done outside the for loop, unless we need to wait until the data is loaded.

Reply:
It probably should only log if the data is loaded (since that's when the person sees it) but only doing so once would be preferred. I'll try a few things on this one.